cmd/applyconfig/applyconfig.go

```diff
@@ -465,6 +467,10 @@ func fileFilter(info os.FileInfo, path string) (bool, error) {
 		return true, nil
 	}
 
+	if ignoreFiles.StringSet().Has(path) {
```

Comment:
```suggestion
	if ignoreFiles.StringSet().Has(filepath.BaseName(path)) {
```

Wouldn't that be more logical?

Reply:
Specially for `cert-manager.yaml`, we have several files with this name in different folders.

Reply:
https://github.com/openshift/release/blob/master/clusters/build-clusters/01_cluster/cert-manager/cert-manager.yaml
and
https://github.com/openshift/release/blob/master/clusters/build-clusters/common/cert-manager.yaml